src/Glpi/Form/Destination/FormDestination.php

```diff
@@ -156,6 +156,10 @@ public static function canPurge(): bool
     #[Override]
     public function canPurgeItem(): bool
     {
+        if ($this->fields['is_mandatory']) {
```

Comment:
I'm always a bit lost with can/cant*Item. Could not that prevent a regular cascade purge? If that's already a tested case (I did not check), it's OK for me.

Reply:
Manual testing seems OK, cascade deletion works without issues.

Reply:
:+1: